x-pack/plugins/fleet/public/index.ts

```diff
@@ -62,9 +64,15 @@ export { PackagePolicyEditorDatastreamPipelines } from './applications/fleet/sec
 export type { PackagePolicyEditorDatastreamPipelinesProps } from './applications/fleet/sections/agent_policy/create_package_policy_page/components/datastream_pipelines';
 export { PackagePolicyEditorDatastreamMappings } from './applications/fleet/sections/agent_policy/create_package_policy_page/components/datastream_mappings';
 export type { PackagePolicyEditorDatastreamMappingsProps } from './applications/fleet/sections/agent_policy/create_package_policy_page/components/datastream_mappings';
-
 export type { DynamicPagePathValues } from './constants';
 
+export type { InputFieldProps } from './applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/components/package_policy_input_var_field';
+
+export const PackagePolicyInputVarField = lazy(() =>
```

Comment:
should we name it `LazyPackagePolicyInputVarField` to indicate that it's lazy?

Reply:
👍